### ISSUE TITLE
Sort photos the same as Photos app

### ISSUE THREAD

**Issue by brockboland** [vokal/VOKMultiImagePicker-iOS#21]
I'm not sure if there's anything we can do about this, but the image picker seems to list the photos in chunks that are internally sorted, but the chunks are not in order. These screenshots might help explain.

In photos:
![image](https://cloud.githubusercontent.com/assets/163838/6983228/f675eea4-d9e0-11e4-90da-7bf1b9771ced.png)

In the image picker:
![image](https://cloud.githubusercontent.com/assets/163838/6983239/162a060e-d9e1-11e4-9205-9f919c1c7930.png)

**Comment by designatednerd** [comment]
@brockboland If you take a look in the "moments" or other folders in the Photos app, is there any kind of logic to how they're sorted in there?

**Comment by brockboland** [comment]
Yup, looks like they're sorted the same there as in the Albums tab

![image](https://cloud.githubusercontent.com/assets/163838/6984190/c4eda6fa-d9eb-11e4-9377-c47c6bd85ff6.png)
